subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -88,6 +97,17 @@ static int get_published_payload(struct mqtt_client *client, u8_t *write_buf,
 	return 0;
 }
 
+/**
+ * @brief Wait for an update to be accepted by AWS for updating the Job
```

Comment:
What does "for updating the Job Execution" mean?

"update ... for updating" is a bit awkward.

Reply:
Hasty written not good documentation. It was intended to replace having a comment in `update_job_execution` so when `update_job_execution` is called we send a request to update to Job Execution with some new information we have to wait until this update is applied (or rejected) before we can do another update to the Job Execution if not our update will be rejected since we are sending the wrong `versionNumber`.

Not sure how to condense it to good documentation but I'll try my best explaining it again.

Reply:
Re-worded